ci/Jenkinsfiles/build.groovy

```diff
@@ -536,10 +538,17 @@ pipeline {
           try {
             parallel stages
           } catch (err) {
-            // TODO NXP-29512: on PR, make the build continue even if there is a test error
+            // TODO NXP-29512: on a PR, make the build continue even if there is a test error
             // on other environments than the dev one
             // to remove when all test environments will be mandatory
-            if (!isPullRequest() || err.message.startsWith("dev:")) {
+            def errorMessage = err.message
+            if (isPullRequest() && !errorMessage.startsWith("dev:")) {
+              echo """
+                Unit tests error: ${errorMessage}
+                Waiting for NXP-29512, on a PR, the build continues even if there is a unit test error in other
+                environments than the dev one.
```

Comment:
```suggestion
                While waiting for NXP-29512 to be resolved, the build in non-dev environment continues even if there are unit test error.
```

Reply:
> on a PR

Cannot remove this, it's representative, look at the actual test in the `if`.

For the the beginning of the sentence, not sure about the added value: suggestion is longer, we try to keep things short.

For the end, it is not that clear: "the build in non-dev environment continues" suggests that only the build that would take place in the dev environment (so the "dev unit tests stage"?) would continue. In fact, we're talking about the whole build.